backend/src/main/java/cookme/recipesmodel/Recipes.java

```diff
@@ -0,0 +1,15 @@
+package cookme.recipesmodel;
+
+import lombok.With;
+
+import java.util.List;
+
+public record Recipes(@With String id,
+                      String name,
+                      String description,
+                      Integer time,
```

Comment:
If you have just minutes its ok, but you are more flexible with double

Reply:
thank you i will change it